product_docs/docs/pgd/5/postgres-configuration.mdx

```diff
@@ -36,7 +36,7 @@ which vary according to the size and scale of the cluster:
     -  One per database on that instance
     -  Four per PGD-enabled database
     -  One per peer node in the PGD group 
-    -  One for each writer-enabled per peer node in the PGD group
+    -  One for each writer-enabled per peer node in the PGD group 
```

Comment:
Should the word "per" in this item be deleted?

Reply:
I don't think so. It's not right though....
It's the number of peer nodes times the (number of writers (bdr.num_writers) plus one).

There's a spreadsheet that has the calculations which has to date defied being written as text.